### ISSUE TITLE
[java] ConsecutiveAppendsShouldReuse false-positive

### ISSUE THREAD

**Issue by cowwoc** [pmd/pmd#4559]
**Affects PMD Version:** 6.55.0

**Rule:** ConsecutiveAppendsShouldReuse

Please provide the rule name and a link to the rule documentation:
https://pmd.github.io/pmd/pmd_rules_java_performance.html#consecutiveappendsshouldreuse

**Description:**
ConsecutiveAppendsShouldReuse flags text blocks as a violation.

**Code Sample demonstrating the issue:**

```java
public class Testcase
{
  public static void main(String[] args)
  {
    StringBuilder out = new StringBuilder();
    out.append("""
      Line1
      Line2
      Line3
      Line4
      Line5
      """
  }
}
```

**Expected outcome:** No violation

PMD reports a violation at line 6, but that's wrong. That's a false positive.

**Running PMD through:** Maven

**Comment by jsotuyod** [comment]
I've rechecked this under PMD 7.0.0-rc3 and the issue doesn't reproduce. Try upgrading.
